### PR TITLE
returnEmptyString parameter

### PR DESCRIPTION
**Pull request by AntoineCAMILLE**
This commit add a feature that allow this component to return an empty string, which means that the field needs to be empty
Sorry for my bad english :/

### CODE REVIEW

File: tests/unit.test.ts [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import {isNumber, formatTimeItem, validateTimeAndCursor} from '../src/index';
+import validate = WebAssembly.validate;
```
Comment (antonfisher) [inline tests/unit.test.ts]: What is this line for? It seems not to be used in the file.

File: tests/component.test.tsx [hunk 2]
```diff
@@ -177,4 +250,12 @@ describe('Component', () => {
     const eventA = {target: {value: 'a', selectionEnd: 1}, persist};
     expect(a.simulate('change', eventA).state('value')).toEqual('12:34');
   });
+
+  test('should handle character deletion (returnEmptyString = true)', () => {
+    const eventC = {target: {value: '12:3', selectionEnd: 4}, persist};
+    expect(c.simulate('change', eventC).state('value')).toEqual('');
+
+    const eventD = {target: {value: '12:34:5', selectionEnd: 4}, persist};
+    expect(d.simulate('change', eventD).state('value')).toEqual('');
```
Comment (antonfisher) [inline tests/component.test.tsx]: As I get it, this test describes the whole point to have `returnEmptyString` parameter. Please, correct me if I am wrong, the original intention is to clean up the time field just by pressing `DELETE` key once?

**Review by antonfisher — CHANGES_REQUESTED** [review]
Hi @AntoineCAMILLE, thanks for using this small component!
Please, take a look at the comments I've left on the PR.

File: tests/unit.test.ts (hunk 1 above)
Comment (AntoineCAMILLE) [reply tests/unit.test.ts]: It's a useless line, i'll change that

File: tests/component.test.tsx (hunk 2 above)
Comment (AntoineCAMILLE) [reply tests/component.test.tsx]: Yes that's it, when the user press `DELETE` key or `BACKSPACE`

**Comment by AntoineCAMILLE** [comment]
Fixed the useless import
I don't understand why the coverage is lowered by 0.2% is it a problem ?

**Pull request closed without merging**